docs/src/modules/spring/pages/using-jwts.adoc

```diff
@@ -0,0 +1,190 @@
+= Using JWTs in Spring
+
+include::ROOT:partial$include.adoc[]
+
+https://jwt.io[JSON Web Tokens] (JWTs) offer a way to authenticate and authorize clients to your services, without requiring anything external to your service to verify the tokens. This is ideal in a serverless environment where your services need to be able to run as autonomously as possible.
+
+A JWT consists of a set of claims and a signature that verifies that these claims were issued by an issuer that you trust. These claims may identify a user, or they may indicate that a user has access to a particular resource, or that a user has a particular role, or is a member of a group, etc.
+
+For example, when a user logs in to a system, they may be given a JWT that contains their username as a claim. When the user makes a request on any of your services, they can present that JWT, and those services can verify that the token is valid, and read the username from the JWT to know that this request was made by that user.
+
+Another common use case is to use claims for authorization. Imagine a social network, one of the services in that network is a friends service. When the user fetches their list of friends from the friends service, it may send them a JWT for each friend in their friends list. For each friend, the corresponding JWT will contain a claim indicating that that user is a friend of the logged in user. When the logged in user wants to send a message to another friend, they can include the JWT of that friend in their request to send a message, and the message service can verify that that JWT has the necessary friendship claim, and allow the message to be sent.
```

Comment:
Since we don't support this for the Spring SDK, we should probably remove this bit.

Reply:
will do.